test/e2e-go/features/devmode/devmode_test.go

```diff
@@ -54,14 +53,15 @@ func TestDevMode(t *testing.T) {
 
 	// 2 transactions should be sent within one normal confirmation time.
 	for i := uint64(0); i < 2; i++ {
-		txn = fixture.SendMoneyAndWait(firstRound+i, 100000, 1000, sender.Address, receiver.String(), "")
-		require.Equal(t, firstRound+i, txn.Txn.Txn.FirstValid)
+		txn = fixture.SendMoneyAndWait(firstRound+i, 100001, 1000, sender.Address, receiver.String(), "")
+		// SendMoneyAndWait subtracts 1 from firstValid
+		require.Equal(t, firstRound+i-1, uint64(txn.Txn.Txn.FirstValid))
```

Comment:
```suggestion
    round := firstRound+i
		txn = fixture.SendMoneyAndWait(round, 100001, 1000, sender.Address, receiver.String(), "")
		// SendMoneyAndWait subtracts 1 from firstValid
		require.Equal(t, round-1, uint64(txn.Txn.Txn.FirstValid))
```
Ignorable nit. Just a bit easier to read.